Assets/Scripts/MiniGames/Song/SongGameController.cs

```diff
@@ -55,40 +130,92 @@ void Start()
         startPlaying = false;
     }
 
-    // Update is called once per frame
-    void Update()
+	void lose_game(float c) 
+	{
+		if (!setmessage) 
+			{
+			if (c > .6f) { failure.text = lose_message[Random.Range(0, lose_message.Count - 1)]; }
```

Comment:
not necessary, but you can use lose_message.RandomElement() which i think is safer and more clear